### PR TITLE
Honest Validator Spec Updates (Excluding Aggregation)

### PR DESCRIPTION
**Pull request by cemozerr**
## PR Description
Sets the attestation data index. Produces attestations 1/3 of a way into slot. Checks if Deposit has MIN_DEPOSIT_AMOUNT. Replaces CommitteAssignment triple with a Class.

### CODE REVIEW

File: services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java [hunk 1]
```diff
@@ -243,7 +243,7 @@ private void onTick(Date date) {
           this.currentEpochGauge.set(compute_epoch_at_slot(nodeSlot).longValue());
           STDOUT.log(Level.INFO, "******* Slot Event *******", ALogger.Color.WHITE);
           STDOUT.log(Level.INFO, "Node slot:                             " + nodeSlot);
-          Thread.sleep(SECONDS_PER_SLOT * 1000 / 2);
+          Thread.sleep(SECONDS_PER_SLOT * 1000 / 3);
```
Comment (ajsutton) [inline services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java:246]: Not for this bit of work, but I've logged https://pegasys1.atlassian.net/browse/BC-101 to replace this type of `Thread.sleep` call with a more event driven timer.  Only really matter down the track when we start thinking through how to catch up if we fell behind.

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/CrosslinkCommittee.java [hunk 2]
```diff
@@ -19,17 +19,17 @@
 
 public class CrosslinkCommittee {
```
Comment (ajsutton) [inline ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/CrosslinkCommittee.java]: Is this the right time to rename this? Crosslinks don't exist anymore so I think this is something like `BeaconCommittee` now? Or maybe just `Committee`.  Happy to leave it for a future PR though.

File: validator/client/src/main/java/tech/pegasys/artemis/validator/client/CommitteeAssignment.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.artemis.validator.client;
+
+import com.google.common.primitives.UnsignedLong;
+import java.util.List;
+
+public class CommitteeAssignment {
```
Comment (ajsutton) [inline validator/client/src/main/java/tech/pegasys/artemis/validator/client/CommitteeAssignment.java:19]: Awesome, this class really does make everything so much easier to understand.

**Review by ajsutton — APPROVED** [review]
LGTM. Nice work.

File: services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java (hunk 1 above)
Comment (cemozerr) [reply services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java:246]: Agreed.

File: ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/CrosslinkCommittee.java (hunk 2 above)
Comment (cemozerr) [reply ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/CrosslinkCommittee.java]: done

File: validator/client/src/main/java/tech/pegasys/artemis/validator/client/CommitteeAssignment.java (hunk 3 above)
Comment (cemozerr) [reply validator/client/src/main/java/tech/pegasys/artemis/validator/client/CommitteeAssignment.java:19]: ty ty ty

**Pull request merged**